book/website/community-handbook/governance/working-groups.md

```diff
@@ -0,0 +1,222 @@
+* **Create a slack channel for collaboration** (not required - but encouraged): Having an ongoing channel makes it easier to communicate across. What other communication channels might this group need?
+* **Add information about working group in Ways of Working and/or Governance documents**: Is there a public and up-to-date record of this working group?
+
+### Approving a Working Group
+
+The approval process for a working group requires a bit more involvement from the wider community and leadership council, as it is important to understand what resources might be available to supp, and/or if the working group's aims are appropriate for the community. 
+
+1. **Completed community charter**: All working groups need to fill out a proposed working group charter ([template here](https://hackmd.io/@aleesteele/template-ttw-wg-charter)) that should be presented to the TTW leadership council and broader community for discusssion and review
+2. **Open discussion and/or Q&A**: All Discuss your idea in community spaces: such as the TTW Slack workspace, Github discussions/issues, and/or Community Calls like the Collaboration Cafe.  
+3. **Leadership council approval**: This creation of this working group must be approved by *The Turing Way* leadership council at their regular organisating calls. These meetings happen every three months, which you can find out about [in this schedule](https://).
+
+### Maintaining a Working Group
+
+While initiating a working group, maintaining a working group's activities is integral for the group's success. 
+
+On a quarterly basis, a working group should revisit its structures and ways of working in order to establish that the group is adhering to its established purpose. The following questions can be guidelines for doing so. These questions have also been compiled into this [working group quarterly audit template]():
+
+* **Review purpose**: Is this working group still fulfilling its established purpose? 
+* **Review progress**: Has this working been able to make progress towards its stated goals? What adjustments may need to be made in order to support this kind of work?
+* **Review process**: Is the existing way of working valuable for this team currently? Are there other processes (decision-making, conflict manage)
+    * Note: The Community Manager and/or Project Manager can help with facilitating this! 
+* **Review roles**: Is everyone in the group in a role they would like to stay in? If not, how should these roles change?
+
+### Sunsetting a Working Group
+
+Deciding when to sunset a working group can be difficult. Often, working groups may continue on indefinitely, and while there are a variety of reasons for this, this may be because there are not means of recognising when a group's aims and/or goals have shifted, or if a group's work is no longer needed. Knowing when it might be time to sunset a working group or shift its aims or direction is an important
```

Comment:
```suggestion
Deciding when to sunset a working group can be difficult. 
Often, working groups may continue on indefinitely, and while there are a variety of reasons for this, this may be because there are not means of recognising when a group's aims and/or goals have shifted, or if a group's work is no longer needed. 
```